src/Compilers/CSharp/Portable/Symbols/Source/SourcePropertySymbol.cs

```diff
@@ -273,6 +273,14 @@ internal sealed class SourcePropertySymbol : PropertySymbol, IAttributeTargetSym
                     _lazyParameters = CustomModifierUtils.CopyParameterCustomModifiers(overriddenOrImplementedProperty.Parameters, _lazyParameters, alsoCopyParamsModifier: isOverride);
                 }
             }
+            else if (_refKind == RefKind.RefReadOnly)
```

Comment:
Filed #20053 for that.

---
In reply to: [119769069](https://github.com/dotnet/roslyn/pull/19658#discussion_r119769069) [](ancestors = 119769069)